### PR TITLE
Update readme to reference docs webpage content

### PR DESCRIPTION
**Pull request by faxu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ ONNX Runtime is up to date and backwards compatible with all operators (both DNN
 
 * [Supported operators/types](./docs/OperatorKernels.md)
   * *Operators not supported in the current ONNX spec may be available as a [Contrib Operator](./docs/ContribOperators.md)*
-* [Extensibility: Add a custom operator/kernel](docs/AddingCustomOp.md)
+* [Extensibility: Add a custom operator/kernel](https://www.onnxruntime.ai/docs/how-to/add-custom-op.html)
```
Comment (xadupre) [inline README.md:72]: I would have written ``(./docs/how-to/AddingCustomOp.md)`` since the link above is working.

File: README.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ Official builds are available on PyPi (Python), Nuget (C#/C/C++), Maven Central
 * Default CPU Provider (Eigen + MLAS)
 * GPU Provider - NVIDIA CUDA
 * GPU Provider - DirectML (Windows)
-  * *On Windows, the [DirectML execution provider](./docs/execution_providers/DirectML-ExecutionProvider.md) is recommended for optimal performance and compatibility with a broad set of GPUs.*
+  * *On Windows, the [DirectML execution provider](https://www.onnxruntime.ai/docs/reference/execution-providers/DirectML-ExecutionProvider.html) is recommended for optimal performance and compatibility with a broad set of GPUs.*
```
Comment (xadupre) [inline README.md:81]: Same comment as above.

File: README.md (hunk 1 above)
Comment (snnn) [reply README.md]: It's in a different branch. So it can't be a relative link.

**Review by snnn — APPROVED** [review]

**Pull request merged**